<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScene</class>
 <widget class="QWidget" name="MainScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>狂野枪战</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resrouces.qrc">
    <normaloff>:/Resource/Icon/GunIcon.png</normaloff>:/Resource/Icon/GunIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="BtnOption">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resource/Icon/OptionBtn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>211</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="BtnStartGame">
      <property name="styleSheet">
       <string notr="true">background:rgb(47, 210, 255);
color:black;
background-color: rgb(161, 161, 161);</string>
      </property>
      <property name="text">
       <string>开始游戏</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnOption2">
      <property name="styleSheet">
       <string notr="true">background:rgb(47, 210, 255);
color:black;
background-color: rgb(161, 161, 161);</string>
      </property>
      <property name="text">
       <string>游戏设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnQuitGame">
      <property name="styleSheet">
       <string notr="true">background:rgb(47, 210, 255);
color:black;
background-color: rgb(161, 161, 161);</string>
      </property>
      <property name="text">
       <string>退出游戏</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>BtnStartGame</zorder>
   <zorder>BtnQuitGame</zorder>
   <zorder>BtnOption2</zorder>
  </widget>
  <widget class="QGraphicsView" name="GameWindow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="BulletCheck">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BloodLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>GAME OVER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RestartButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>320</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>重开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>410</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="BackGround">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resource/Image/BackGround/MainBackGround.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnRestart">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resource/Icon/RestartIcon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>771</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Resource/Image/BackGround/Title.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>BackGround</zorder>
  <zorder>GameWindow</zorder>
  <zorder>BloodLabel</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>BtnOption</zorder>
  <zorder>BulletCheck</zorder>
  <zorder>label</zorder>
  <zorder>RestartButton</zorder>
  <zorder>QuitButton</zorder>
  <zorder>BtnRestart</zorder>
  <zorder>Title</zorder>
 </widget>
 <resources>
  <include location="Resrouces.qrc"/>
 </resources>
 <connections/>
</ui>
